packages/web/src/components/payout-wallet-modal/PayoutWalletModal.tsx

```diff
@@ -245,7 +245,10 @@ export const PayoutWalletModal = () => {
                 })
 
               await sdk.services.solanaRelay.relay({
-                transaction
+                transaction,
+                confirmationOptions: {
+                  commitment: 'finalized'
```

Comment:
i think we may be able to get away with confirmed

Reply:
Yeah most likely, but I'd rather be safe.. not all the DNs need to use the same RPCs and I'm worried about some being behind etc.

Reply:
I think based on probabilities, confirmed is enough for us and i would rather have one periodically fail than have a user wait like 60s on this page. going to do confirmed for now and we can revisit next week when you're back in office

Reply:
I'm more concerned with the ACDC indexing side not succeeding in checking the account existing because that RPC is behind than a revert on the front end, which would mean all purchases would fail to index on some nodes. Maybe we should send the current slot with the ACDC transaction?

Reply:
setting the payout wallet would just fail in this case and the user would look like they don't have a payout wallet set. I think that is acceptable behavior in the bad case versus making the happy path take 60s. can discuss your proposal here next week!

Reply:
Is that true? I don't think it's guaranteed that all nodes will pass or fail the token account check uniformly. But yeah confirmed seems fine

Reply:
good point. the slot move it probably right, and we should probably do something like that for other things.
even w/ finalized, a given RPC provider could be behind or something... so I don't totally trust it in either case.